app/src/ui/branches/pull-request-list.tsx

```diff
@@ -195,12 +197,32 @@ export class PullRequestList extends React.Component<
         dispatcher={this.props.dispatcher}
         repository={pr.base.gitHubRepository}
         onDropOntoPullRequest={this.onDropOntoPullRequest}
+        onContextMenu={this.onPullRequestItemContextMenu}
         onMouseEnter={this.onMouseEnterPullRequest}
         onMouseLeave={this.onMouseLeavePullRequest}
       />
     )
   }
 
+  private onPullRequestItemContextMenu = (
+    event: React.MouseEvent<HTMLDivElement>
+  ): void => {
+    event.preventDefault()
+
+    const items = generatePullRequestContextMenuItems({
+      pr: this.props.selectedPullRequest,
+      onViewPullRequestOnGitHub: this.OnViewPullRequestOnGitHub,
+    })
+
```

Comment:
```suggestion
    event.preventDefault()
    
    const {selectedPullRequest: pr } = this.props
    const onViewPullRequestOnGitHub =  pr !== null ? this.OnViewPullRequestOnGitHub : undefined;
    
    const items = generatePullRequestContextMenuItems({onViewPullRequestOnGitHub})
```

For both the `generateBranchContextMenuItems` and the `generateBranchContextMenuItems`, it seems we are unnecessarily passing down the PR object. This is more of nit. It looks to me we could have a little cleaner code if we conditionally pass the method since that is already being checked for definition?

(Example above may make the linter unhappy, it is just to show what I mean 😄)